VERSION

```diff
@@ -1 +1 @@
-0.9.1
+0.9.2
```

Comment:
this is reserved for when we cut a new release of the project and since this PR is just meant to upgrade the golang version, could you please revert this change?